dotCMS/src/integration-test/java/com/dotcms/content/elasticsearch/business/ESIndexAPITest.java

```diff
@@ -37,9 +48,77 @@ public void test_createIndex_newIndexShouldHaveProperReplicasSetting() throws IO
 
             Assert.assertEquals("0-all", replicasSetting);
         } finally {
-            APILocator.getESIndexAPI().delete(APILocator.getESIndexAPI()
-                    .getNameWithClusterIDPrefix(newIndexName));
+            esIndexAPI.delete(esIndexAPI.getNameWithClusterIDPrefix(newIndexName));
         }
     }
 
+    @Test
+    public void testGetIndicesStatsWhenStatsTypeIsLongShouldPass(){
+
+        final Map<String, Object> jsonMap = new HashMap<>();
+
+        final Map<String, Object> indices = new HashMap<>();
+        final Map<String, Object> indexDetails = new HashMap<>();
+        final Map<String, Object> statsMap = new HashMap<>();
+        final Map<String, Object> countMap = new HashMap<>();
+        final Map<String, Object> sizeMap = new HashMap<>();
+
+        countMap.put("count", Long.valueOf(5));
+        sizeMap.put("size_in_bytes", Long.valueOf(2000));
+
+        statsMap.put("docs", countMap);
+        statsMap.put("store", sizeMap);
+
+        indexDetails.put("primaries", statsMap);
+        indices.put("myDummyIndex", indexDetails);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String literal "myDummyIndex" appears 4 times in this file; the first occurrence is on line 73](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5178104)